<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_D3D11Class</class>
 <widget class="QMainWindow" name="QT_D3D11Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT_D3D11</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>光照</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="cBtnSpec">
         <property name="text">
          <string>高光</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioWhiteLight">
         <property name="text">
          <string>白光</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBlueLight">
         <property name="text">
          <string>蓝光</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioRedLight">
         <property name="text">
          <string>红光</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="rBtnNormal">
         <property name="text">
          <string>普通</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnWater">
         <property name="text">
          <string>水面</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnFire">
         <property name="text">
          <string>火焰效果</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnShadow">
         <property name="text">
          <string>阴影效果</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>模型显示</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="DxWindow" name="W_DxWindow" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>模型文件</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="FileText">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>5</red>
                <green>239</green>
                <blue>13</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>5</red>
                <green>239</green>
                <blue>13</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>光照方向</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="lightSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>相机方向</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="camSlider">
         <property name="value">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>207</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="BtnModify">
         <property name="text">
          <string>修改模型</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionTex"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionNew"/>
    <addaction name="actionTex"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="QT_D3D11.qrc">
     <normaloff>:/image/Resources/open3.bmp</normaloff>:/image/Resources/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="QT_D3D11.qrc">
     <normaloff>:/image/Resources/new2.bmp</normaloff>:/image/Resources/new2.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建模型</string>
   </property>
  </action>
  <action name="actionTex">
   <property name="icon">
    <iconset resource="QT_D3D11.qrc">
     <normaloff>:/image/Resources/open3.bmp</normaloff>:/image/Resources/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>选择贴图</string>
   </property>
   <property name="toolTip">
    <string>贴图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DxWindow</class>
   <extends>QWidget</extends>
   <header location="global">DxWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QT_D3D11.qrc"/>
 </resources>
 <connections/>
</ui>
